app/controllers/collections_controller.php

```diff
@@ -122,6 +124,11 @@ public function of($username, $correctnessLabel = null, $lang = null)
         $this->set('corpus', $corpus);
         $this->set('username', $username);
         $this->set('correctness', $correctness);
+        if(empty($userId)) {
+            $this->set("userExists", false);
+            return;
+        }
+        $this->set("userExists", true);
```

Comment:
I think it makes more sense to `return` as soon as you computed `$userId`. There is no need to call `$this->UsersSentences->getPaginatedCorpusOf()` with an invalid `$userId`.